redwood-layout-dom/src/main/kotlin/app/cash/redwood/layout/dom/HTMLElementRedwoodLayoutWidgetFactory.kt

```diff
@@ -46,6 +47,8 @@ class HTMLElementRedwoodLayoutWidgetFactory(
       direction = "row",
       overflowSetter = { style.overflowX = it },
     )
+
+  override fun Spacer(): Spacer<HTMLElement> = TODO("Not yet implemented")
```

Comment:
I think this should just be a pretty basic <div> with an explicit width and height.

I'm not sure if the TODO GitHub action will pick up on TODO function calls or not.

Reply:
I'll add it in!